README.md

```diff
@@ -10,6 +10,13 @@ examples get run and tailor the output.
     gem install rspec-core # for rspec-core only
     rspec --help
 
+  Want to run against the `master` branch? You'll need to include the dependent
+  RSpec repos as well. Add the following to your `Gemfile`:
+
+    %w[rspec rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
```

Comment:
Also, `rspec` isn't needed...the gem itself is empty.  It's just a meta gem that allows you to run `gem install rspec` (or put `gem 'rspec'` in your Gemfile) and get all the gems.  If you're individually specifying all the gems (as is the case here), then you don't need the rspec metagem.